apps/cyberstorm-remix/app/styles/settingsItem.css

```diff
@@ -58,4 +58,18 @@
     border-radius: var(--radius-md);
     background: var(--color-surface-default);
   }
+
+  @media (width <= 48rem) {
```

Comment:
Did you check these work as intended on all the other views using this component?